src/http.rs

```diff
@@ -308,6 +308,14 @@ pub enum Auth {
         /// The AWS service name to use for signing.
         service: String,
     },
+
+    /// Custom Authorization Header Value, will be inserted into the headers as `Authorization: < value >`
+    CustomAuthorization {
```

Comment:
Rename this to preserve UX consistency: 
```suggestion
    Custom {
```